topostats/run_topostats.py

```diff
@@ -180,9 +182,11 @@ def process_scan(
     gaussian_mode : str
         Mode for filtering (default is 'nearest').
     otsu_threshold_multiplier : Union[int, float]
-
         Factor by which lower threshold is to be scaled prior to masking.
     background : float
+        The value to average the background around.
```

Comment:
Thanks for adding the description here as well as the docstring for `zrange` :+1:

Reply:
No problem! I forgot that when we moved it in the config I'd have to add it to grains too.